### PR TITLE
Change block bundle major number

### PR DESCRIPTION
**Pull request by core23**
There will be a new major version soon

### CODE REVIEW

**Comment by OskarStark** [comment]
So waiting until released right?

File: config/projects.yml [hunk 1]
```diff
@@ -55,14 +55,12 @@ block-bundle:
...
-      php: ['7.1', '7.2', '7.3', '7.4snapshot']
+    4.x:
+      php: ['7.3', '7.4snapshot']
       target_php: '7.3'
       versions:
-        symfony: ['3.4']
-        sonata_core: ['3']
-        # This has to be resolved: https://travis-ci.org/sonata-project/SonataBlockBundle/jobs/130298942#L230
-        #sonata_admin: ['3']
+        symfony: ['4.3']
```
Comment (OskarStark) [inline config/projects.yml]: Needed? 🤔

Comment (core23) [reply config/projects.yml]: As soon as we have a 4.4 release, yes

**Comment by core23** [comment]
Duplicate of #523

**Comment by greg0ire** [comment]
Woops sorry I didn't notice this issue

**Pull request closed without merging**